### PR TITLE
DG: fix TOC rendering on website

### PR DESCRIPTION
**Pull request by bnjmnt4n**
| Before | After |
|---|---|
| image | image |

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -2,7 +2,8 @@
 layout: page
 title: Developer Guide
 ---
-* Table of Contents
+### Table of Contents
```
Comment (bnjmnt4n) [inline docs/DeveloperGuide.md:5]: Not sure about making this a heading, but I thought it made more sense than leacing it as a bullet point. Wdyt?

Comment (stanley-1) [reply docs/DeveloperGuide.md:5]: Nice!

**Review by stanley-1 — APPROVED** [review]
Thanks for patching this up!

**Review by EssWhyy — APPROVED** [review]
Nice this looks much better, thanks for fixing the formatting!

**Pull request merged**